src/components/home/home.jsx

```diff
@@ -30,21 +30,24 @@ class HomePage extends React.Component {
 
   render() {
     return <React.Fragment>
+
```

Comment:
FYI. Careful of whitespace changes. Just so there's less stuff for reviewers to look at.